core/src/window_service.rs

```diff
@@ -100,15 +116,12 @@ impl WindowServiceMetrics {
                 self.num_errors_cross_beam_recv_timeout,
                 i64
             ),
+            ("num_addrs", self.addrs.len(), i64),
+            ("top_addr_1", packed_ips.next().unwrap_or(0), i64),
+            ("top_addr_2", packed_ips.next().unwrap_or(0), i64),
+            ("top_addr_3", packed_ips.next().unwrap_or(0), i64),
```

Comment:
```suggestion
            ("top_addr_decimal_1", packed_ips.next().unwrap_or(0), i64),
            ("top_addr_decimal_2", packed_ips.next().unwrap_or(0), i64),
            ("top_addr_decimal_3", packed_ips.next().unwrap_or(0), i64),
```

a vain attempt to remind us that doing stats on these in chronograf is pointless

Reply:
How about `top_addr_as_int_1`?